### PR TITLE
Update documentation to point to new snapshot and release repositories

### LINKED ISSUE #500: Add link to snapshot repo in CONTRIBUTING.md
Update this section to refer to the snapshot repo instead of downloading from CI: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/CONTRIBUTING.md#snapshot-builds

### PR DESCRIPTION
**Pull request by iNikem**
Closes #500

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,28 +4,37 @@
...
 The instrumentation agent is enabled using the `-javaagent` flag to the JVM.
-Configuration parameters are passed as Java system properties (`-D` flags) or
-as environment variables. Both the Java agent and exporter configuration must
-be defined before the application JAR. For example:
+```
+java -javaagent:path/to/opentelemetry-auto-<version>-all.jar \
+     -jar myapp.jar
+```
+By default OpenTelemetry Java agent uses 
+[OpenTelemetry exporter](https://github.com/open-telemetry/opentelemetry-java/tree/master/exporters/otlp)
```
Comment (trask) [inline README.md:28]: "OpenTelemetry exporter" sounds ambiguous to me, leaves me wondering "which OpenTelemetry exporter?"

```suggestion
[OTLP exporter](https://github.com/open-telemetry/opentelemetry-java/tree/master/exporters/otlp)
```

File: README.md [hunk 2]
```diff
@@ -227,9 +240,6 @@ High-level roadmap:
       [opentelemetry-specification#522](https://github.com/open-telemetry/opentelemetry-specification/issues/522))
   * See issues with label
     [specification](https://github.com/open-telemetry/opentelemetry-java-instrumentation/labels/specification)
-* Simpler exporter configuration
-  * See issue
-    [#370](https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/370)
```
Comment (trask) [inline README.md:232]: 🎉

File: README.md [hunk 3]
```diff
@@ -4,28 +4,37 @@
...
 
-Download the [latest
-release](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases)
-of the Java agent and available exporters.
+[ ![Download](https://api.bintray.com/packages/open-telemetry/maven/opentelemetry-java-instrumentation/images/download.svg) ](https://bintray.com/open-telemetry/maven/opentelemetry-java-instrumentation/_latestVersion)
+
+The easiest way to start is to download "all-in" package, which includes instrumentation agent,
+instrumentations for all supported libraries and all available data exporters.
+This provides completely automatic out of the box experience.
+TODO add here screenshot from Bintray download list.
```
Comment (trask) [inline README.md:20]: This seems a bit complicated. Can we provide a direct link to download the `-all` package?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -71,7 +71,7 @@ following "clusters" of classes:
 
 - `auto-tooling-and-instrumentation.isolated/` - contains `agent-tooling`
 module and `instrumentation` submodules, loaded and isolated inside
-`AgentClassLoader`. Including OpenTelemetry SDK.
+`AgentClassLoader`. Including OpenTelemetry SDK and data exporters.
```
Comment (trask) [inline CONTRIBUTING.md:74]:
```suggestion
`AgentClassLoader`. Including OpenTelemetry SDK (and the built-in exporters when using the `-all` artifact).
```

File: README.md (hunk 3 above)
Comment (iNikem) [reply README.md:20]: The best I can do now is https://www.jfrog.com/confluence/display/BT/Managing+Uploaded+Content#ManagingUploadedContent-CreatingandEditingaDownloadList. Have a link in documentation to the latest version's screen and there is "Download list" which has our `-all` artefact. I asked JFrog if we can have direct link to the latest version of a specific file.

**Pull request merged**